packages/features/ee/workflows/components/WorkflowStepContainer.tsx

```diff
@@ -515,6 +549,29 @@ export default function WorkflowStepContainer(props: WorkflowStepProps) {
                   )}
                 </>
               )}
+              {form.getValues(`steps.${step.stepNumber - 1}.action`) !== WorkflowActions.SMS_ATTENDEE && (
+                <Button
+                  type="button"
+                  className="mt-7 w-full"
+                  disabled={isTestActionDisabled}
+                  onClick={() => {
+                    if (
+                      form.getValues(`steps.${step.stepNumber - 1}.action`) !== WorkflowActions.SMS_NUMBER
+                    ) {
+                      testActionMutation.mutate({
+                        action: step.action,
+                        emailSubject: step.emailSubject || "",
+                        reminderBody: step.reminderBody || "",
+                        template: step.template,
+                      });
+                    } else {
+                      setConfirmationDialogOpen(true);
```

Comment:
Asking for additional confirmation if user really wants to send SMS to the given number